scubagoggles/rego/Commoncontrols.rego

```diff
@@ -599,33 +664,58 @@ NonCompliantOUs4_1 contains {
     not IsGoodLimit(LastEvent.NewValue)
 }
 
-tests contains {
-    "PolicyId": "GWS.COMMONCONTROLS.4.1v0.3",
+# The following checks for web session duration less than or equal to 12 hours.
+# Not sure whether this really fits with 4.1 requirement.  The 12 hours is
+# mentioned, but the requirement deals with forced re-authentication after the
+# session expiration - is this automatic with session expiration?  If so, then
+# this check is probably appropriate.  It seems to fit with the event log
+# method above.
+
+NonCompliantOUs4_1 contains {
+    "Name": OU,
+    "Value": sprintf("Web session duration: %s", [duration])
+}
+if {
+    multipliers := {"s": 1, "m": 60, "h": 3600}
+    # This is the requirement limit for session duration:
+    webSessionMax := 12 * multipliers["h"]
+    some OU, settings in input.policies
+    duration := settings.security_session_controls.webSessionDuration
```

Comment:
This another instance where the policy API returns a fixed value, `1209600s`, or two weeks. But oddly enough _just_ for the primary OU, it seems to correctly handle the child OUs.

Reply:
After implementing Google's policy "reduction" process, this is working for me
when I change settings in the Admin UI.